test/integration/backward_compatible/serial/CompactRestartTest.js

```diff
@@ -54,6 +54,10 @@ describe('CompactRestartTest', function() {
         if ((await TestUtil.compareServerVersionWithRC(RC, '5.2.0')) >= 0 && !TestUtil.isClientVersionAtLeast('5.2.0')) {
             this.skip();
         }
+        if ((await TestUtil.compareServerVersionWithRC(RC, '5.2.0')) < 0) {
```

Comment:
Can you reuse `await TestUtil.compareServerVersionWithRC(RC, '5.2.0')` by assigning it to a constant?

Reply:
Done.